recipes/georasters/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "georasters" %}
+{% set version = "0.5.10" %}
+{% set sha256 = "86ad677d76ff71879f7bc6a5aad0d2f60ed826107d5fd2d4705105b8fb4ef648" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed
```

Comment:
Missing a dot at the end of this (you need to tell pip to install from "here"). (see the example on the cf webpage)

Reply:
Indeed. Thx!